prdoc/pr_6689.prdoc

```diff
@@ -0,0 +1,19 @@
+title: '[pallet-revive] Update gas encoding'
+doc:
+- audience: Runtime Dev
+  description: |-
+    Update the current approach to attach the `ref_time`, `pov` and `deposit` parameters to an Ethereum transaction.
+    Previously we will pass these 3 parameters along with the signed payload, and check that the fees resulting from `gas x gas_price` match the actual fees paid by the user for the extrinsic.
```

Comment:
Previously, these three parameters were passed along with the signed payload, and the fees resulting from `gas × gas_price` were checked to ensure they matched the actual fees paid by the user for the extrinsic ?

Reply:
yes, anything unclear here?